### ISSUE TITLE
Gradle init always greates subprojects and this is not deactivatable

### ISSUE THREAD

**Issue by andreasgottardi** [gradle/gradle#16516]
I used to create single projects with "gradle init" and there was no "lib" folder present. instead the whole project was just a single project that got imported into Eclipse correctly as one project. Since a few releases back "gradle init" for type java-library with dsl Groovy Gradle always greates a multi project which is very annoying because I always have to rework the folder structure before I can import it into Eclipse. If I don't do that Eclipse imports two projects and this IDE is not suitable for multi project structures.

### Expected Behavior
There should be a command line parameter to disable the multi project build. There is the parameter
```
--split-project     Split functionality across multiple subprojects?
```
which enables it, but it is enabled by default. It should maybe be
```
--no-split-project
```
to disable the behavior.

### Current Behavior
"gradle init" always creates sub projects and this can not be deactivated. The generated project should be reworked in order to get a single project.

### Context
I create a lot of projects because our company moves to a Gradle based project structure and every time I create a project I have to rework the folder.

### Steps to Reproduce 
Create a project with the following command line:
```
gradle init --type java-library --test-framework junit-jupiter --dsl kotlin --project-name gradletest --package gradletest
```
Then try to import is into Eclipse with Buildship and watch how two projects get imported. There is no option to disable this behavior.

### Your Environment
I am working on Windows and Linux (both x64) with the latest Azul systems OpenJDK 15.

**Comment by donat** [comment]
I understand how generating multi-project builds with the `init` plugin can be controversial. At the same time, can you elaborate on what's not working in Buildship in this scenario? There might be name clashes for the subproject names, but Gradle should automatically deduplicate the overlapping names.

**Comment by donat** [comment]
This is still relevant.

**Comment by andreasgottardi** [comment]
Sorry, didn't want to cause any troubles and I wrote a script to init projects that removes the generated sub project folder and the corresponding line from the settings file.

**Comment by donat** [comment]
Don't worry about it, you did not cause any trouble. I'm happy you could solve your issue.

**Comment by jnehlmeier** [comment]
It seems like nowadays this old single project layout is obsolete. At least gradle documentation does not talk about it anymore at all. It only describes the case above, having a root project and a single or multiple sub project(s).

**Comment by andreasgottardi** [comment]
Yeah, but it's okay. Even if creating a project from scratch it is just a short rework of the project. And a new project is not created every day.
